frontend/src/components/LocationSearch/LocationSearchInput.tsx

```diff
@@ -54,6 +57,11 @@ export const LocationSearchInput = (props: LocationSearchInputProps): JSX.Elemen
   const showLoadingIndicator = searchInput !== "" && (foundLocations.type === "outdated" || inputStillInDebounce);
   const showResults = searchInput !== "" && (foundLocations.type === "success" || foundLocations.type === "outdated");
 
+  const applyLocationName = (): void => {
+    setSearchInput(props.searchName ?? "");
```

Comment:
remove line 34 as the state is now stored higher up.

Reply:
Omitted due to time constraints